src/shared/proto/tbots_robot_status_msg.proto

```diff
@@ -0,0 +1,154 @@
+syntax = "proto3";
```

Comment:
This just seems renamed. Is that all that changed?

Reply:
yikes that's from a bad merge